packages/sit-onyx/src/components/OnyxDataGrid/features/index.ts

```diff
@@ -219,15 +219,41 @@ export const useDataGridFeatures = <
     header: { component: HeaderCell },
     cell: { component: (props) => String(props.modelValue) },
   };
+
+  const numberFormatter = (value: TEntry[keyof TEntry] | undefined) => {
+    if (typeof value !== "number") {
```

Comment:
```suggestion
    if (typeof value !== "number" || typeof value === "bigint") {
```

The Intl. formatter also supports bigint

Reply:
now tries to render any given value